build/ci/jenkins_pipeline_build

```diff
@@ -1,4 +1,11 @@
 pipeline {
+    parameters {
+        string(name: 'IMAGE_VERSION', defaultValue: "1.7.0")
+        string(name: 'OPERATOR_IMAGE', defaultValue: "ibm-block-csi-operator-amd64")
```

Comment:
I'm not sure this parameter should have a default value (until we move to multiarch images)